ProcessMaker/Http/Controllers/Api/ProcessController.php

```diff
@@ -381,6 +382,7 @@ public function update(Request $request, Process $process)
         // Catch errors to send more specific status
         try {
             $process->saveOrFail();
+            $process->assignAssetsToProjects($request, Process::class);
```

Comment:
1. When I access `/processes/{id}/edit` to update the properties of the process, the "Projects" select box does not pre-select the projects to which the process already belongs. (I think this would also happen in the other assets (scripts, screens, etc.).
2. When updating the properties of the process and selecting one or multiple projects, repeated assets are generated and I end up with the same asset repeated within the project.

Reply:
Good catch!

Reply:
Created two observation tickets to handle this separately from this PR. 

https://processmaker.atlassian.net/browse/FOUR-10124
https://processmaker.atlassian.net/browse/FOUR-10125

Marking as resolved.